lib/renderer.js

```diff
@@ -188,7 +187,7 @@ class Tokenizer extends MarkedTokenizer {
     const cap = rules.inline.text.exec(src);
     if (cap) {
       let text;
-      if (this.lexer.state.inRawBlock) {
+      if (this.lexer.state.inRawBlock || this.rules.inline.url.exec(src)) {
```

Comment:
@curbengh @SukkaW 
Would you please check this?
I rebased this branch to [main](https://github.com/hexojs/hexo-renderer-marked/tree/568e5e90d430251eac8c21bb6d83d128275b7ee9) and add this `or` condition.

Tests are passed. I believe this change is no problem.